test/service/test_azul_webservice.py

```diff
@@ -0,0 +1,221 @@
+import os
+import sys
+import logging
+import requests
+from threading import Thread
+from unittest import TestCase
+from urllib.parse import urljoin
+from chalice.local import LocalDevServer
+from chalice.config import Config
+
+sys.path.append(os.path.join(os.environ['AZUL_HOME'], 'lambdas', 'service'))
+
+from app import app as service_app
+
+
+log = logging.Logger('azul-indexer-tests')
+stream_handler = logging.StreamHandler(sys.stdout)
+log.addHandler(stream_handler)
+log.setLevel(logging.DEBUG)
+
+TEST_PROTOCOL = "http"
+TEST_DOMAIN = "localhost"
+TEST_PORT = 54632
+TEST_HOST = f"{TEST_PROTOCOL}://{TEST_DOMAIN}:{TEST_PORT}"
+
+URL_REQUEST_TIMEOUT = 4
+
+
+class ChaliceServerThread(Thread):
+    def __init__(self, app, config, host, port):
+        super().__init__()
+        self.server_wrapper = LocalDevServer(app, config, host, port)
+
+    def run(self):
+        self.server_wrapper.serve_forever()
+
+    def kill_thread(self):
+        self.server_wrapper.server.shutdown()
+        self.server_wrapper.server.server_close()
+
+
+BAD_FACET_HTTP_ERROR_CODE = 400
+
+FILTER_FACET_MESSAGE = {"Code": "BadRequestError",
+                        "Message": "BadRequestError: Unable to filter by undefined facet bad-facet."}
+SORT_FACET_MESSAGE = {"Code": "BadRequestError",
+                      "Message": "BadRequestError: Unable to sort by undefined facet bad-facet."}
+
+
+class ChaliceRequestTestCase(TestCase):
+    def setUp(self):
+        log.debug(f"(Test {__name__}): Setting up tests")
```

Comment:
Remove `__name__` from the message it will be logged automatically if you instantiate the logger like I suggested above. You could add self._testMethodName to the log message.